lib/commands/diff/index.js

```diff
@@ -36,6 +36,10 @@ module.exports = function (config, cli) {
   }
 
   var diff = child.spawn("git", ["diff", lastCommit, "--color", filePath]);
+  var pagerOpts = {
+    pager: "less",
+    args: ["-R"] // parse raw control characters
+  };
```

Comment:
One problem with `process.env.PAGER` is that not everyone will have it, contrary to the expectation of `default-pager`. Inheriting `stdio` does not have this problem.